### ISSUE TITLE
Viewing commits.

### ISSUE THREAD

**Issue by royale1223** [EddieRingle/hubroid#90]
It would be awesome to view commits and diff.

**Comment by EddieRingle** [comment]
I agree. This is something I miss from 2.0 and will have fun re-implementing it for 3.0.

**Comment by royale1223** [comment]
Great. Looking forward. 

_Sent via Hubroid_
